vaadin-progress-bar.html

```diff
@@ -13,15 +13,12 @@
   <template>
     <style>
       :host {
-        height: 3px;
-      }
-
-      [part="bar"] {
-        background: #ddd;
+        height: 0.5em;
```

Comment:
I would rather have this on the host element, so it’s easy to override. Do the inner parts adapt to the height of the host, btw?

Reply:
Aw, fff. I’m stupid, I read the diff completely wrong. Sorry. Nevermind 😆